problem-1/problem-1.mojo

```diff
@@ -1,22 +1,14 @@
-from python import Python
-
-def lengthOfLastWord(enterword: String):
```

Comment:
I'm trying to understand why you changed the defining keyword here from "def" to "fn". I mean it worked fine with def, any reason for that?

Reply:
fn in mojo allow to use types, which enable the compiler to optimize more effectively.